pallets/kensetsu/src/migrations.rs

```diff
@@ -256,6 +256,19 @@ pub mod v1_to_v2 {
                 );
                 weight += <T as frame_system::Config>::DbWeight::get().writes(1);
 
+                StablecoinInfos::<T>::insert(
+                    AssetIdOf::<T>::from(KGOLD),
```

Comment:
I don't remember, is KGOLD already registered as asset in some previous migration?